data/campaigns/Eastern_Invasion/utils/zSort_abilities.cfg

```diff
@@ -0,0 +1,557 @@
+    [/event]
+    [event]
+        name=die,last breath,terror_despawn #last breath makes this trigger before mal-ravanal's very tedious death animation
+        first_time_only=no
+
+        [filter]
+            ability=terror
+            [or]
+                id=Mal-Ravanal
+            [/or]
+        [/filter]
+
+        [remove_item]
+            image=terror_aura
+        [/remove_item]
+        [remove_object]
+            object_id=terrified
+        [/remove_object]
+    [/event]
+
+    #------------------------
+    # TERORR-UNIT MOVES
+    #------------------------
+    [event]
+        name=moveto
+        first_time_only=no
+
+        [filter]
+            ability=terror
+        [/filter]
+
+        {VISUALIZE_TERROR $unit.x $unit.y}
+        [fire_event]
+            name=terror_refresh
+        [/fire_event]
+    [/event]
+
+    [event]
+        name=exit hex
+        first_time_only=no
+
+        [filter]
+            ability=terror
+        [/filter]
+
+        [remove_item]
+            image=terror_aura
+        [/remove_item]
+    [/event]
+
+    #------------------------
+    # VICTIM MOVES
+    #------------------------
+    [event]
+        name=moveto
+        first_time_only=no
+
+        [filter_condition]
+            [have_unit]
+                ability=terror
+            [/have_unit]
+        [/filter_condition]
+
+        [fire_event]
+            name=terror_refresh
+        [/fire_event]
+        [on_undo]
+            [fire_event]
+                name=terror_refresh
+            [/fire_event]
+        [/on_undo]
+        [allow_undo]
+        [/allow_undo]
+    [/event]
+
+    [event]
+        name=terror_refresh
+        first_time_only=no
+        [remove_object]
+            [filter]
+                [not]
+                    [filter_location]
+                        radius=2
+
+                        [filter]
+                            ability=terror
+                        [/filter]
+                    [/filter_location]
+                [/not]
+            [/filter]
+
+            object_id=terrified
+        [/remove_object]
+        {APPLY_TERROR ([filter_location]
+        radius=2
+
+        [filter]
+            ability=terror
+        [/filter]
+    [/filter_location])}
+[/event]
+#enddef
+
+#define ABILITY_SHROUD
+    [resistance]
+        id=shroud
+        name= _ "shroud"
+        description= _ "While defending, this unit has 50% fire and arcane resistance."
+        affect_self=yes
+        active_on=defense
+        value,max_value=50,50
+        apply_to=fire,arcane
+    [/resistance]
+#enddef
+
+# this is most effective on high defense terrain (castles, villages, mountains, etc), which fits with the theme of the ability
+#define ABILITY_FORTIFY
+    [dummy]
+        id=fortify
+        name= _ "fortify"
+        description= _ "When defending, adjacent allies are 10% less likely to be hit (including by magical attacks)."
+    [/dummy]
```

Comment:
This can i think also be implemented as `[change_to_hit] [affect_adjacent]` and `apply_to=opponent`

Reply:
Ack, fortify isn't even used in the current version.  Should be fine to delete this altogether.